### PR TITLE
WIP: Added basic form for adding payment method

### PR DESCRIPTION
**Pull request by a8uhnf**
* added payment form using v3 JS SDK
* need to discuss

### CODE REVIEW

**Comment by sadlil** [comment]
https://developers.braintreepayments.com/guides/drop-in/javascript/v3
> The latest version of our Drop-in UI is currently in beta

So are we sure, we are bumping version for ui? @TamalSaha
Without dropin-ui there will be no validation and the magical things that was there before - like the list cards and things - unless we do this manually.

File: js/views/lobby/new-payment-method.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/* global ENTER_KEY, ESC_KEY */
```
Comment (sajibcse68) [inline js/views/lobby/new-payment-method.js]: cleanup

File: js/views/lobby/new-payment-method.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    console.log('Hello World next To new payment form page', data, $('#second-form').serializeArray());
+  },
+  submitToBraintree(e) {
+    e.preventDefault();
+    this.getClientToken();
+  },
+  getClientToken() {
+    apiClient.billing.v1beta1.paymentMethodService.userClientToken()
+      .then(apiLib.checkStatus)
+      .then(this.nextStep.bind(this));
```
Comment (sajibcse68) [inline js/views/lobby/new-payment-method.js]: What about calling `this.braintreeCall.bind(this)` directly?

**Comment by TamalSaha** [comment]
I did not notice that v3 UI is in beta. In that case let's stick to v2. We already know how to use that.

**Pull request closed without merging**